### PR TITLE
Sync WD EC docs with latest CnC schema

### PR DESCRIPTION
**Pull request by scot-taylor-ibm**

### CODE REVIEW

File: schema.md [hunk 1]
```diff
@@ -342,12 +357,39 @@ The schema is arranged as follows.
         - `party`: A string that identifies the party to whom the sentence applies.
       - `provenance_ids`: An array of one or more hashed values that you can send to IBM to provide feedback or receive support.
     - `categories`: An array that lists the functional categories into which the element falls; in other words, the subject matter of the element.
-      - `label`: A string that lists the identified category. You can find a list of [categories](/docs/services/compare-comply?topic=compare-comply-contract_parsing#contract_categories) in [Understanding contract parsing](/docs/services/discovery?topic=discovery-contract_parsing#contract_parsing).
+      - `label`: A string that lists the identified category. You can find a list of [categories](/docs/services/compare-comply?topic=compare-comply-contract_parsing#contract_categories) in [Understanding element classification](/docs/services/compare-comply?topic=compare-comply-contract_parsing).
```
Comment (NickyBleiel) [inline schema.md]: I don't think we are supposed to point them to the C&C docs.

File: schema.md [hunk 2]
```diff
@@ -342,12 +357,39 @@ The schema is arranged as follows.
...
       - `provenance_ids`: An array of one or more hashed values that you can send to IBM to provide feedback or receive support.
     - `attributes`: An array that identifies document attributes. Each object in the array consists of three elements:
-      - `type`: The type of attribute. Possible values are `Currency`, `DateTime`, `DefinedTerm`, `Duration`, `Location`, `Number`, `Organization`, `Percentage`, and `Person`.
+      - `type`: The type of attribute. Possible values are `Currency`, `DateTime`, `Duration`, `Location`, `Number`, `Organization`, `Percentage`, and `Person` as described at [Attributes](/docs/services/compare-comply?topic=compare-comply-contract_parsing#attributes).
```
Comment (NickyBleiel) [inline schema.md]: Same as above.

File: schema.md [hunk 3]
```diff
@@ -395,10 +437,13 @@ The schema is arranged as follows.
...
-        - `type`: The type of attribute. Possible values are `Address`, `Currency`, `DateTime`, `Location`, `Organization`, and `Person`.
+        - `type`: The type of attribute. Possible values are `Address`, `Currency`, `DateTime`, `Duration`, `Location`, `Number`, `Organization`, `Percentage`, and `Person`.
         - `text`: The text that is associated with the attribute.
         - `location`: The location of the attribute as defined by its `begin` and `end` indexes.
-    - `key_value_pairs`: An array that specifies any key-value pairs in tables in the input document.
+    - `contexts`: An array of objects that list text that is related to the table contents and that precedes or follows the current table. Each object contains the following elements:
+      - `text`: The related text.
+      - `location`: The location of the related text as defined by its `begin` and `end` indexes in the input document.
+    - `key_value_pairs`: An array that specifies any key-value pairs in tables in the input document. For more information, see [Understanding key-value pairs](/docs/services/compare-comply?topic=compare-comply-understanding_tables#key-value-pairs).
```
Comment (NickyBleiel) [inline schema.md]: Same

File: schema.md (hunk 1 above)
Comment (scot-taylor-ibm) [reply schema.md]: Fixed.

File: schema.md (hunk 3 above)
Comment (scot-taylor-ibm) [reply schema.md]: Fixed.

File: schema.md (hunk 2 above)
Comment (scot-taylor-ibm) [reply schema.md]: Fixed.

**Pull request merged**